convert2rhel/unit_tests/cert_test.py

```diff
@@ -70,20 +78,53 @@ def test_get_cert_path_missing_cert(self):
     @unit_tests.mock(utils, "DATA_DIR", unit_tests.NONEXISTING_DIR)
     @unit_tests.mock(system_info, "arch", "nonexisting_arch")
     def test_get_cert_path_nonexisting_dir(self):
-        self.assertRaises(SystemExit, cert.SystemCert._get_cert_path)
+        self.assertRaises(SystemExit, cert.SystemCert._get_cert)
         self.assertEqual(len(cert.loggerinst.critical_msgs), 1)
 
-    @unit_tests.mock(cert.SystemCert, "_system_cert_dir", unit_tests.TMP_DIR)
     @unit_tests.mock(tool_opts, "arch", "x86_64")
     @unit_tests.mock(utils, "DATA_DIR", os.path.join(base_data_dir, "6", "x86_64"))
     def test_install_cert(self):
         # By initializing the cert object we get a path to an existing
         # certificate based on the mocked parameters above
         system_cert = cert.SystemCert()
+        system_cert._target_cert_dir = unit_tests.TMP_DIR
 
         system_cert.install()
 
-        cert_filename = os.path.basename(system_cert._cert_path)
-        installed_cert_path = os.path.join(system_cert._system_cert_dir, cert_filename)
+        installed_cert_path = os.path.join(system_cert._target_cert_dir, system_cert._cert_filename)
         self.assertTrue(os.path.exists(installed_cert_path))
         shutil.rmtree(unit_tests.TMP_DIR)
+
+
+@pytest.mark.parametrize(
+    ("directory", "filename"),
+    ((unit_tests.TMP_DIR, "existing.file"), (unit_tests.TMP_DIR, unit_tests.NONEXISTING_FILE), (None, None)),
+)
+def test_remove(monkeypatch, directory, filename):
+    if filename == "existing.file":
+        try:
+            utils.mkdir_p(directory)
+            os.mknod(os.path.join(directory, filename))
+        except FileExistsError:
+            pass
+
+    monkeypatch.setattr(cert.SystemCert, "_get_cert", value=mock.Mock(return_value=("anything", "anything")))
+    if directory is not None or filename is not None:
```

Comment:
it is enough to write just:
```suggestion
    if directory or filename:
```